doc/proxy-metrics.md

```diff
@@ -133,3 +135,92 @@ request_total{
 
 [prom-format]: https://prometheus.io/docs/instrumenting/exposition_formats/#format-version-0.0.4
 [pod-template-hash]: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#pod-template-hash-label
+
+# Transport-Level Metrics
+
+The following metrics are collected at the level of the underlying transport
+layer.
+
+### `connection_opens`
+
+A counter of the total number of transport connections which have been opened.
+
+### `connection_closes`
+
+A counter of the total number of transport connections which have been closed.
+
+### `connections`
+
+A gauge of the number of transport connections currently open.
+
+### `sent_bytes`
+
+A counter of the total number of sent bytes.
+
+### `received_bytes`
+
+A counter of the total number of recieved bytes.
+
+### `connection_duration_ms`
+
+A histogram of the duration of the lifetime of a connection, in milliseconds.
+
+### `connection_sent_bytes`
+
+A histogram of the number of bytes sent over the lifetime of a connection.
+This is updated when a connection closes.
+
+### `connection_recieved_bytes`
+
+A histogram of the number of bytes recieved over the lifetime of a connection.
+This is incremented when a connection closes.
+
+## Labels
+
+Each of these metrics has the following labels:
+
+* `direction`: `inbound` if the connection was to/from the proxy's Inbound router,
```

Comment:
This is a bit tautological.  I'm not sure exactly how to rephrase but inbound represents connections established from outside the pod to the proxy and from the proxy to the application.  Outbound represents connections established from the application to the proxy and from the proxy to outside of the pod.